test/ttmlir/Dialect/TTIR/select/select_decomposition_tests.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: ttmlir-opt --ttir-to-ttir-decomposition %s | FileCheck %s
```

Comment:
Thanks for adding this test @rpavlovicTT! Can we just move this decomposition one under the newly created folder in TTIR to have all decomposition tests in one place? Something like:
TTIR\Decompositions\...

FYI @LPanosTT

Reply:
I’m going to be adding this directory very soon to #1332 and likely merging it later today

Reply:
Done